client/src/main/resources/templates/js/main.js

```diff
@@ -524,15 +524,15 @@ function displaySource(src, title) {
     }
 
     if(getPageSize(pageAdditionalId) >= rtFilterSearch.length) {
-      $(pageAdditionalId + 'testcase-page-info').html("Showing all rows");
+      $(pageAdditionalId + "testcase-page-info").html("Showing all rows");
     } else {
-      $(pageAdditionalId + 'testcase-page-info').html("Showing " + rtIndex + " to " + endIndex + " of " + rtFilterSearch.length + " rows");
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Line is too long.](https://www.codacy.com/app/SeLion/paypal-SeLion/file/1307322591/issues/source?bid=2928425&fileBranchId=2951451#l529)